stock_getter/frontend_stockgetter/scripts/card.js

```diff
@@ -184,10 +263,63 @@ class Card {
             name: 'Close',
             x: data[0],
             y: data[4],
-            line: {color: '#f0187d'}
+            mode: 'markers',
+            marker: {
+                color: '#f0187d',
+                size: 2
+            }
+          }
+          
+        // Handle forecast case
+          let plotData = []
+          if (this.forecast) {
+            let trace4 = {
+                type: "scatter",
+                mode: "lines",
+                name: 'High Forecast',
+                x: data[0],
+                y: data[7],
+                line: {
+                    color: '#17BECF',
+                    width: 1
+                }
+              }
+              
+              let trace5 = {
+                type: "scatter",
+                mode: "lines",
+                name: 'Low Forecast',
+                x: data[0],
+                y: data[8],
+                line: {
+                    color: '#7F7F7F',
+                    width: 1
+                }
+              }
+              
+              let trace6 = {
+                type: "scatter",
+                mode: "lines",
+                name: 'Close Forecast',
+                x: data[0],
+                y: data[9],
+                line: {
+                    color: '#f0187d',
+                    width: 1
+                }
+              }
+              plotData = [trace1,trace2, trace3, trace4, trace5, trace6];
+          
+            } else {
+              plotData = [trace1,trace2, trace3];
+          }
+
+          let legPositon;

```

Comment:
Why not simplify as:
const legPosition = this.forecast ? 0.05 : 0.35

Reply:
We can, but i don't like ternary expressions. They are less clear.